nucypher/policy/conditions/_utils.py

```diff
@@ -66,3 +76,51 @@ def _deserialize_condition_lingo(data: Union[str, Dict[str, str]]) -> Union['Ope
     lingo_class = _resolve_condition_lingo(json_data=data)
     instance = lingo_class.from_dict(data)
     return instance
+
+
+def evaluate_conditions_for_ursula(lingo, context, log, ursula):
```

Comment:
Is the `ursula` parameter needed here?

Reply:
It's unused however, I was considering including some ursula-specific metadata (checksum, connected chains, etc.) in log messages.  Do you think that adds any debugging value?

Reply:
Interesting thought. I don't know if they do right now... 

If you think not right now as well we can rename this function to just be `evaluate_conditions`. Dealer's choice.